app/jobs/application_job.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class ApplicationJob < ActiveJob::Base
+  sidekiq_options retry: 1
```

Comment:
question: why not `retry: 0` or even `retry: false`? won't setting it to 1 let the jobs to be retried once?

Reply:
Yes, just once. My thinking on this was that we "might as well" let it do just one because of "network jitter" potential. There's a chance something just blanks out quickly and comes back quickly. But, I woudn't want it to go more than once I think. Basically thinking like "ok let's just keep retries alive and see what we see" kind of thing. Kind of an experiment. I do think we've handled most of the cases ourselves in terms of looking for problematic responses from the web service, but, maybe there are legit ones that could be solved by a quick retry? I guess I'm just second guessing the idea of complete elimination of retries for now - mainly because it doesn't hurt to try once more, there is no cost to the user and no cost in complexity.

Reply:
my concern is about the use case we thought while we were discussing to disable the retries. Like for example if the user place tried to update location on a hold but then for some reason the job failed, then user went ahead and tried again was able to change the location but this time choose another location. Then the previous job retry kicked in and changed the location to the previous chose of user's. I guess the scenario was sth like this 🤔 When does the retry happen?

Reply:
Yes, this is a concern for sure. I was thinking that because the first retry happens within only a second or two that would mitigate the issue. So in the above scenario you'd have a user sitting there looking at a spinner waiting for the thing to finish, which would only be a few more seconds tacked on. 

Starting to gravitate back to just turning it off completely again though, who knows how long the initial timeout or whatever uncaught error will take to be produced and then retried on.

Reply:
changed in latest